docs-conceptual/azps-3.8.0/get-started-azureps.md

```diff
@@ -6,76 +6,105 @@ ms.author: sttramer
 manager: carmonm
 ms.devlang: powershell
 ms.topic: get-started-article
-ms.date: 01/17/2020
+ms.date: 04/24/2020
 ---
 
 # Get started with Azure PowerShell
 
-Azure PowerShell is designed for managing and administering Azure resources from the command line. Use Azure PowerShell when you want to build automated tools that use the Azure Resource Manager model.
-Try it out in your browser with [Azure Cloud Shell](/azure/cloud-shell/overview), or install on your
-local machine.
+Azure PowerShell is designed for managing and administering Azure resources from the command line.
+Use Azure PowerShell when you want to build automated tools that use the Azure Resource Manager
+model. Try it out in your browser with [Azure Cloud Shell](/azure/cloud-shell/overview), or install
+on your local machine.
 
 This article helps you get started with Azure PowerShell and teaches the core concepts behind it.
 
 ## Install or run in Azure Cloud Shell
 
-The easiest way to get started with Azure PowerShell is by trying it out in an Azure Cloud Shell environment.
-To get up and running with Cloud Shell, see [Quickstart for PowerShell in Azure Cloud Shell](/azure/cloud-shell/quickstart-powershell).
-Cloud Shell runs PowerShell 6 on a Linux container, so Windows-specific functionality isn't available.
+The easiest way to get started with Azure PowerShell is by trying it out in an Azure Cloud Shell
+environment. To get up and running with Cloud Shell, see
+[Quickstart for PowerShell in Azure Cloud Shell](/azure/cloud-shell/quickstart-powershell). Cloud
+Shell runs PowerShell on a Linux container, so Windows-specific functionality isn't available.
 
-When you're ready to install Azure PowerShell on your local machine, follow the instructions in [Install the Azure PowerShell module](install-az-ps.md).
+When you're ready to install Azure PowerShell on your local machine, follow the instructions in
+[Install the Azure PowerShell module](install-az-ps.md).
 
 ## Sign in to Azure
 
-Sign in interactively with the `Connect-AzAccount` cmdlet. Skip this step if you use Cloud Shell: Your Azure Cloud Shell session is already authenticated
-for the environment, subscription, and tenant that launched the Cloud Shell session.
+Sign in interactively with the
+[Connect-AzAccount](/powershell/module/az.accounts/connect-azaccount) cmdlet. Skip this step if you
+use Cloud Shell. Your Azure Cloud Shell session is already authenticated for the environment,
+subscription, and tenant that launched the Cloud Shell session.
 
 ```azurepowershell-interactive
 Connect-AzAccount
 ```
 
-If you're in a non-US region, use the `-Environment` parameter to sign in. Get the name of the environment for your region by using
-the [Get-AzEnvironment](/powershell/module/Az.Accounts/Get-AzEnvironment) cmdlet. For example, to sign in to Azure China 21Vianet:
+If you're in a non-US region, use the **Environment** parameter to sign in. Get the name of the
```

Comment:
I would not use "if you're in a non-US region", the environment is made to target the cloud this is not Azure Public. What about "If you need to use a national cloud, use ..." ?